### PR TITLE
the great purple purge of 2016

### PR DESCRIPTION
**Pull request by medghaim**

### CODE REVIEW

**Comment by medghaim** [comment]
#3399CC is link, both hover and focus.

**Comment by JZ6** [comment]
rebase with this please
https://github.com/JZ6/PoGoMap/commit/03ebd9b9acaf77178e7696d095838f5378c86b2e

**Comment by JZ6** [comment]
actually i can fix it

File: css/all.css [hunk 1]
```diff
@@ -29,7 +29,7 @@ p{font-family:"Source Sans Pro", sans-serif;}
 .nav-opener:hover{opacity:.9;}
 .nav-active .nav-opener:after, .nav-active .nav-opener:before{-moz-transform:rotate(45deg);-webkit-transform:rotate(45deg);-ms-transform:rotate(45deg);transform:rotate(45deg);top:16px;left:5%;right:5%;}
 .nav-active .nav-opener:after{-moz-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);-ms-transform:rotate(-45deg);transform:rotate(-45deg);}
-.visual{min-height:904px;position:relative;overflow:hidden;text-align:center;color:#8d81ac;font-size:0;letter-spacing:-4px;padding:104px 0 0;background-image:url(../images/capture.png);background-size:cover}
+.visual{min-height:904px;position:relative;overflow:hidden;text-align:center;color:#e0e0e0;font-size:0;letter-spacing:-4px;padding:104px 0 0;background-image:url(../images/capture.png);background-size:cover}
```
Comment (JZ6) [inline css/all.css]: remove this

**Pull request merged**